custom_components/tahoma/const.py

```diff
@@ -66,6 +66,7 @@
     "ThermalEnergySensor": SENSOR,
     "VenetianBlind": COVER,
     "WaterDetectionSensor": BINARY_SENSOR,  # widgetName, uiClass is HumiditySensor (sensor)
+    "WaterHeatingSystem": CLIMATE,
```

Comment:
```suggestion
    "AtlanticPassAPCDHW": CLIMATE, # widgetName, uiClass is WaterHeatingSystem (not supported)
```

Please register this device based on the widgetName, otherwise all `WaterHeatingSystem` devices will show up in Home Assistant, even those they are not supported.